cmdstanpy/stanfit.py

```diff
@@ -485,12 +488,8 @@ def column_names(self) -> Tuple[str, ...]:
         and quantities of interest. Corresponds to Stan CSV file header row,
         with names munged to array notation, e.g. `beta[1]` not `beta.1`.
         """
-        if not self._validate_csv and self._metadata is None:
-            self._logger.warning(
-                'csv files not yet validated, run method validate_csv_files()'
-                ' in order to retrieve sample metadata.'
-            )
-            return None
+        if self._metadata is None:
```

Comment:
Even if we don't go the extra step of calling validate_csv_files in the constructors, I think these changes are great -- unexpectedly returning None can be a real headache